src/main/resources/templates/categories.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="https://www.thymeleaf.org" lang="en">
```

Comment:
what the heck is up with this

Reply:
Fixed with https://github.com/2020-Spring-Cohort/reviews-site-models/pull/11/commits/9e8714340cde9ea75a0862e996807b0abe75cf4e